File: planning_poker_api/src/main/java/com/agiletools/estimo/dtos/UserMessageDto.java
package com.agiletools.estimo.dtos;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class UserMessageDto {
    private Long userId;
    private String userFullName;
    private Long storyId;
    private Long sessionId;
    private String estimation;
    private LocalDateTime votedTime;
    private boolean hasVoted;
    private Integer position;
    private String role;
}

File: frontend/src/utils/dtos/UserMessageDto.ts
import {UserSessionRole} from "../Enums";

export interface UserMessageDto {
  userId: number;
  userFullName: string;
  storyId: number;
  sessionId: number;
  estimation: string;
  votedTime: string;
  hasVoted: boolean;
  position: number;
  role: UserSessionRole;
}
